lib/cli/commands/reset.js

```diff
@@ -30,14 +30,10 @@ const reset = async (packageDir, logFile, options) => {
   }
 
   await run(
-    async () => wpcli(`core config \
-    --dbhost=db \
-    --dbname=wordpress \
-    --dbuser=root \
-    --dbpass='' \
-    --locale=en_US \
-    --extra-php <<PHP
-define( 'FS_METHOD', 'direct' );
+    async () => wpcli(
+      'core config --dbhost=db --dbname=wordpress --dbuser=root --dbpass="" --locale=en_US --extra-php',
+      logFile,
+      `define( 'FS_METHOD', 'direct' );
```

Comment:
Instead of using heredoc to pipe the php code into the command, I added a new `stdin` parameter to both `cli` and `wpcli`. They use the pipe feature of `shelljs`